src/Modules/OrchardCore.Commerce/Migrations/ProductMigrations.cs

```diff
@@ -21,6 +23,10 @@ public int Create()
     {
         _contentDefinitionManager
             .AlterPartDefinition(nameof(ProductPart), builder => builder
+                .WithField("ProductImage", field => field
```

Comment:
Use `nameof(ProductPart.ProductImage)`, instead of a string literal.
Or better yet, use the strongly typed [`AlterPartDefinition<TPart>`](https://github.com/Lombiq/Helpful-Libraries/blob/57171a77a08b1a7ee055bc2167322188ccccde6d/Lombiq.HelpfulLibraries.OrchardCore/Contents/ContentDefinitionManagerExtensions.cs#L33) extension method from HL.